src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -92,11 +99,11 @@
 @Logged
 public class ArmSubsystem extends SubsystemBase {
 
-  private final TalonFX elevatorMotorLeader = new TalonFX(DEVICE_ID_ELEVATOR_MOTOR_LEADER, CANIVORE_BUS_NAME);
+  final TalonFX elevatorMotorLeader = new TalonFX(DEVICE_ID_ELEVATOR_MOTOR_LEADER, CANIVORE_BUS_NAME);
```

Comment:
Why aren't these private?

Reply:
I was trying to write a unit test for `moveToPosition` so I made them package private, but it didn't work. I'll make them private again.